src/components/pages/DatasetsList.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import ReactRouterPropTypes from 'utils/react-router-prop-types';
+import PropTypes from 'prop-types';
+import { Search, Eye } from 'react-feather';
+
+const DatasetItem = (props) => {
+  const {
+    name, description,
+  } = props;
+
+  return (
+    <tr className="intro-x w-full">
+      <td className="w-4/5">
+        <h2 className="text-xl font-medium whitespace-no-wrap">
+          {name}
+        </h2>
+        {description}
+      </td>
+      <td className="w-1/5 align-middle">
+        <div className="text-center">
+          <button type="button" className="button button--md w-full flex items-center lg:justify-around md:justify-center sm:justify-center mr-1 mb-2 bg-theme-1 text-white">
+            <span className="md:hidden sm:hidden lg:inline-block">Переглянути</span><span><Eye /></span>
```

Comment:
Use data list button from midone templates

Reply:
And create component for this button

Reply:
+